exporter/src/main/java/io/zeebe/monitor/SimpleMonitorExporter.java

```diff
@@ -211,19 +214,26 @@ public void open(final Controller controller) {
   }
 
   private void createTables() {
-    try (final Statement statement = connection.createStatement()) {
-
-      final InputStream resourceAsStream =
-          SimpleMonitorExporter.class.getResourceAsStream(CREATE_SCHEMA_SQL_PATH);
-      final String sql =
-          new BufferedReader(new InputStreamReader(resourceAsStream))
-              .lines()
-              .collect(Collectors.joining(System.lineSeparator()));
-
-      log.debug("Create tables:\n{}", sql);
-      statement.executeUpdate(sql);
-    } catch (final Exception e) {
-      throw new RuntimeException(e);
+    if (configuration.createSchema != null && !configuration.createSchema.equals("-")) {
```

Comment:
Why we need to check for not equal to "-" ? The tables are only created if they are not exist anyway I think this is not necessary.

I would like to auto detect via the jdbc url whether it is a mysql/mariadb or other (like h2) instead of this new configuration property.

Reply:
The reason for "-" is to have the ability to skip executing script entirely (and create tables manually outside, as a different user for example). As for autodetection, I don`t have a readymade solution for that - I don`t want to introduce another library, nor I want to do it manually. 

Maybe disable createSchema execution is all we need?
I use docker `/docker-entrypoint-initdb.d` SQL script for example.

Reply:
> Maybe disable createSchema execution is all we need?

I think you're right. The best would be to have simply the option to skip the creation. So the user can start and setup the database by himself. Then it is not necessary to have the mysql dependencies and the extra script I would say.